man/os-release.xml

```diff
@@ -309,6 +309,22 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CODENAME=</varname></term>
+
+        <listitem><para>
+        A lower-case string (no spaces or other characters outside of
+        0–9, a–z, ".", "_" and "-") identifying the operating system
+        release code name, excluding any OS name information or
+        release version, and suitable for processing by scripts or
+        usage in generated filenames. This field is optional and may
+        not be implemented on all systems.
```

Comment:
Shouldn't this be called VERSION_CODENAME= or so? To match VERSION= and VERSION_ID=?